index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/5116129a4d.js" crossorigin="anonymous"></script>
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="styles.css"/>
+</head>
+<body>
```

Comment:
- [ ] As you can see in the image below, your content is overflowing from the max width of the main `body` or `html` element (blue region). This usually happens when the size of one of the elements on your page is greater than the total screen size which causes the whole website to skew to be able to fit that particular element.

> ![image](https://user-images.githubusercontent.com/90556221/185382792-b4b26158-f703-49fa-b18b-71470bc3da39.png)

In your case it is the `<p>` tag which you can see below

> ![image](https://user-images.githubusercontent.com/90556221/185383123-1d47ef20-9c78-43e8-b1e2-4994cec989c4.png)

This usually happens when you use fixed values for the `margin` `padding` or `width` of an element and that fixed value is greater than the screen size. Let's say I'm viewing the website on a mobile 370px wide. If the `<p>` tag above has a width of 400px, my mobile device can't fit that so it will skew the entire website and make it smaller. Kindly alter this code and decrease the width to resolve this issue